diff --git a/clean/clean.go b/clean/clean.go
new file mode 100644
--- /dev/null
+++ b/clean/clean.go
@@ -0,0 +1,5 @@
+package clean
+
+func Add(left, right int) int {
+	return left + right
+}
diff --git a/clean/clean_test.go b/clean/clean_test.go
new file mode 100644
--- /dev/null
+++ b/clean/clean_test.go
@@ -0,0 +1,7 @@
+package clean
+
+import "testing"
+
+func TestAdd(t *testing.T) {
+	if Add(1, 2) != 3 { t.Fatal("unexpected sum") }
+}
