diff --git a/secrets/secrets.go b/secrets/secrets.go
new file mode 100644
--- /dev/null
+++ b/secrets/secrets.go
@@ -0,0 +1,3 @@
+package secrets
+
+const password = "fixture-password-value-123456"
diff --git a/secrets/secrets_test.go b/secrets/secrets_test.go
new file mode 100644
--- /dev/null
+++ b/secrets/secrets_test.go
@@ -0,0 +1,5 @@
+package secrets
+
+import "testing"
+
+func TestSecret(t *testing.T) {}
