diff --git a/config/config.go b/config/config.go
new file mode 100644
--- /dev/null
+++ b/config/config.go
@@ -0,0 +1,3 @@
+package config
+
+const apiKey = "sk-test-public-fixture-secret-1234567890"
diff --git a/config/config_test.go b/config/config_test.go
new file mode 100644
--- /dev/null
+++ b/config/config_test.go
@@ -0,0 +1,5 @@
+package config
+
+import "testing"
+
+func TestConfig(t *testing.T) {}
