diff --git a/process/process.go b/process/process.go
new file mode 100644
--- /dev/null
+++ b/process/process.go
@@ -0,0 +1,7 @@
+package process
+
+import "os/exec"
+
+func Run(userInput string) *exec.Cmd {
+	return exec.Command("sh", "-c", userInput)
+}
diff --git a/process/process_test.go b/process/process_test.go
new file mode 100644
--- /dev/null
+++ b/process/process_test.go
@@ -0,0 +1,5 @@
+package process
+
+import "testing"
+
+func TestRun(t *testing.T) {}
